pkg/controller/controller.go

```diff
@@ -193,6 +193,13 @@ func (ctrl *CSIAttachController) syncVA() {
 	ctrl.handler.SyncNewOrUpdatedVolumeAttachment(va)
 }
 
+func (ctrl *CSIAttachController) shouldProvision(pv *v1.PersistentVolume) bool {
```

Comment:
`shouldProvision` is not a great name. Perhaps `shouldProcessPV`? Not great either, but does not imply provisioning.

Reply:
@jsafrane how about `shouldAttach`? do you have any suggestions?

Reply:
`syncPV` is there only to add / remove finalizers, attachment is controller by `VolumeAttachment` queue. `shouldAttach` does not make sense. `processFinalizers`?